### DISCUSSION TITLE
Repository Exclusion from Hacktoberfest 2024

### DISCUSSION THREAD

**Discussion by saloni-jaiswal-dev** [Saloni6111/DSA_Problems-Hacktoberfest-2024#181] [Announcements]
Hello contributors,

The repository "DSA_Problems-Hacktoberfest-2024" has been excluded from Hacktoberfest 2024. I am currently investigating the issue to resolve it as soon as possible. In the meantime, please avoid contributing to this repo for Hacktoberfest purposes. I’ll provide an update once everything is sorted out. Thank you for your understanding!

**Comment by sagar98cyber** [comment]
Maybe we could help! What is the issue? Any idea? Did you receive any email regarding the same?

**Reply by sagar98cyber** [reply]
I read somewhere on the INTERNET. If there are too many comments (more than LOCs)  GIT considers that as spam.
